app/controllers/v1/people_controller.rb

```diff
@@ -1,7 +1,7 @@
 class V1::PeopleController < V1::BaseController
 
   def create
-    @person = PersonWithRemoteFields.find_or_build(person_params)
+    @person = PersonConstructor.new(person_params).build
```

Comment:
Oh, and I don't agree with you on avoiding `create` as a method name if that is what we want to accomplish. I would agree that we got lazy above validating those creations and not returning good error messages, but we should use accurate terms to describe what we want to accomplish.  In these sorts of instances, I generally adopt the rails convention of appending a exclamation point to differentiate between raising errors or not.  I agree it's not the best approach, but it is a good generic convention that I think a lot of devs adpot and would understand if they were picking up this project later.